Tools/MediaDriverTools/UMDPerfProfiler/Readme.txt

```diff
@@ -0,0 +1,19 @@
+Introduction
+    The purpose of UMD Perf Profiler is to collect the performance data of media tasks, which can help us to know the workloads on GPU side. You can get the media tasks' timing, FPS and concurrency between GPU engines. 
+
+Usage
+• Step1: Build and install the latest internal-release version driver
```

Comment:
What's the mean about  "latest internal-release version driver" ?

Reply:
That's mean we should build the driver with internal-release flag. There are three choices when build the driver, release/internal-release/debug.  The "latest" driver will include lots of bug fixes, that's always the best choice for us :)